src/java.base/share/classes/java/net/SocketImpl.java

```diff
@@ -78,15 +78,15 @@ static <S extends SocketImpl & PlatformSocketImpl> S createPlatformSocketImpl(bo
     public SocketImpl() { }
 
     /**
-     * Creates either a stream or a datagram socket.
+     * Creates a stream socket.
      *
      * @apiNote
-     * The {@link Socket} constructors to create a datagram socket
-     * are deprecated for removal. This method will be re-specified
-     * in a future release to not support creating datagram sockets.
+     * The {@code stream} parameter provided a way in early JDK releases
+     * to create a {@link Socket} that used a datagram socket.
+     * It is no longer possible to do that and therefore this method will
```

Comment:
"It is no longer possible to do that ...", can you try changing this to "The Socket API no longer provides a way to do this so the create method will ways to be called with a stream value of true". I think it might be a bit easier to read.

Reply:
Done